kafka-connect-s3/src/main/java/io/confluent/connect/s3/storage/S3OutputStream.java

```diff
@@ -196,7 +199,9 @@ private MultipartUpload newMultipartUpload() throws IOException {
         newObjectMetadata()
     ).withCannedACL(cannedAcl);
 
-    if (SSEAlgorithm.KMS.toString().equalsIgnoreCase(ssea)
+    if (StringUtils.isNotBlank(ssea) && StringUtils.isNotBlank(sseKey)) {
```

Comment:
The `StringUtils.isNotBlank(ssea)` check here seems weak. Should this be `SSEAlgorithm.AES256.toString().equalsIgnoreCase(ssea)`? I'm not sure exactly what the SDK requires here.

Reply:
Thanks, good point, I will double check but remember reading that only AES256 is supported, I'll make this a stronger check.

Reply:
Yes, only AES256 is supported: https://docs.aws.amazon.com/AmazonS3/latest/dev/ServerSideEncryptionCustomerKeys.html